test/integration/transactionsAPITests.coffee

```diff
@@ -177,8 +188,8 @@ describe "API Integration Tests", ->
       it  "should add a transaction and truncate the large request body", (done) ->
         td = JSON.parse JSON.stringify transactionData
         td.channelID = channel._id
-        td.request.body = ''
-        td.request.body += '1234567890' for i in [0...2*1024*1024]
+        reqBody = largeBody
+        td.request.body = reqBody
```

Comment:
I think we can just use 'largeBody' variable here and remove the variable declaration above